cypress/e2e/consortia/inventory/instance/instance-data-is-not-lost-when-promoting-source-folio-instance.cy.js

```diff
@@ -0,0 +1,183 @@
+import moment from 'moment';
+import uuid from 'uuid';
+import Affiliations, { tenantNames } from '../../../../support/dictionary/affiliations';
+import Permissions from '../../../../support/dictionary/permissions';
+import InventoryInstances from '../../../../support/fragments/inventory/inventoryInstances';
+import ConsortiumManager from '../../../../support/fragments/settings/consortium-manager/consortium-manager';
+import TopMenu from '../../../../support/fragments/topMenu';
+import DateTools from '../../../../support/utils/dateTools';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+// import InventorySearchAndFilter from '../../../../support/fragments/inventory/inventorySearchAndFilter';
```

Comment:
Please remove comments if they're not nessesary

Reply:
I've deleted